src/routes/landing/components/__tests__/Heading.spec.js

```diff
@@ -1,17 +0,0 @@
-// @flow
-
-import React from 'react'
-
-import Heading from '../Heading'
-import { mount } from 'enzyme'
-import { ThemeProvider } from 'styled-components'
-import theme from '../../../../modules/theme/constants/theme'
-
-jest.mock('react-i18next')
-
-describe('Heading', () => {
-  it('should render', () => {
-    const component = mount(<ThemeProvider theme={theme}><Heading /></ThemeProvider>)
-    expect(component).toMatchSnapshot()
-  })
```

Comment:
Testing, whether `Heading` renders successfully is beneficial I guess. Maybe not using snapshot but maybe instead checking wheter some crucial components are rendered

Reply:
Not sure whether this would be beneficial. Right now the test was not useful at least.